compiler/rustc_macros/src/query.rs

```diff
@@ -272,6 +272,34 @@ fn process_modifiers(query: &mut Query) -> QueryModifiers {
                 if desc.is_some() {
                     panic!("duplicate modifier `desc` for query `{}`", query.name);
                 }
+                if query.doc_comments.is_empty() {
```

Comment:
Perhaps you could add a comment here explaining what this branch is doing.

Reply:
Good idea, done.